### PR TITLE
PMM-11751: Verify pmm-server collects data after disconnect

### PR DESCRIPTION
**Pull request by nazarovvi**

### CODE REVIEW

File: tests/pages/dashboardPage.js [hunk 1]
```diff
@@ -1178,6 +1178,7 @@ module.exports = {
     refreshIntervalOption: (interval) => locate(`//*[@role="menuitemradio" and text()="${interval}"]`),
     clickablePanel: (name) => `//section[@aria-label="${name} panel"]//a`,
     dashboardTitle: (name) => locate('span').withText(name),
+    metricPanelNa: (name) => `//section[@aria-label="${name} panel"]//span[text()="N/A"]`,
```
Comment (beatahandzelova) [inline tests/pages/dashboardPage.js:1181]:
```suggestion
    metricPanelNa: (name) => `//section[@aria-label="${name}"]//span[text()="N/A"]`,
```

File: tests/configuration/verifyPMMServerDisconnect_test.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    await I.verifyCommand('docker stop pmm-server-disconnect');
+    I.wait(180);
+    await I.verifyCommand('docker start pmm-server-disconnect');
+    I.wait(60);
+    await I.amOnPage(withCustomBaseUrl(`${dashboardPage.mySQLInstanceOverview.clearUrl}?orgId=1&from=now-2m&to=now-1m`));
+    I.dontSeeElement(dashboardPage.fields.metricPanelNa('Services panel'));
+  },
+);
+
+xScenario(
```
[the remark this answers is not in the document]
Comment (nazarovvi) [reply tests/configuration/verifyPMMServerDisconnect_test.js]: Unskipped and fixed

**Review by beatahandzelova — DISMISSED** [review]

**Review by puneet0191 — APPROVED** [review]

File: tests/configuration/verifyPMMServerDisconnect_test.js [hunk 3]
```diff
@@ -0,0 +1,69 @@
+Feature('Pmm Server stability');
+
+// Address of PMM to be disconnected.
+const pmmServerPort = '8180';
+const basePmmUrl = `http://127.0.0.1:${pmmServerPort}/`;
+let clientServerNetwork = 'pmm-ui-tests_server-network';
+
+BeforeSuite(async ({ I }) => {
+  await I.verifyCommand('docker-compose -f docker-compose-disconnect.yml up -d pmm-server-disconnect');
```
[the remark this answers is not in the document]
Comment (nazarovvi) [reply tests/configuration/verifyPMMServerDisconnect_test.js]: Added

**Comment by nazarovvi** [comment]
https://pmm.cd.percona.com/blue/organizations/jenkins/pmm2-ui-tests/detail/pmm2-ui-tests/31826/pipeline/

**Review by puneet0191 — APPROVED** [review]

**Pull request merged**